js/app/demo/app.jsx

```diff
@@ -15,7 +15,7 @@ define(['react', 'jsx!editor', 'jsx!messages', 'jsx!fixedCode', 'jsx!configurati
                 return result;
             }
 
-            return {
+            var initialState = JSON.parse(window.localStorage.getItem('linterDemoState')) || {
```

Comment:
Merging states would probably be a bit better, but will cause issues when options for a given rule change.